frontend/src/app/account/profile/profile.component.html

```diff
@@ -0,0 +1,207 @@
+<div class="wrapper">
+    <div class="container w-full sm:w-22rem">
+        <h2 class="mt-0 align-self-start">Profile</h2>
+
+        <div class="profile-container">
+            <div class="profile-field">
+                <div class="flex flex-row align-items-center gap-2">
+                    <i class="pi pi-user"></i>
+                    <label for="current username">Username</label>
+                </div>
+                <input
+                    pInputText
+                    id="current username"
```

Comment:
An ID attribute's value must not contain [ASCII whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace#in_html) characters.

Reply:
Changed to ```"currentUsername"``` and ```"currentEmail"```